### PR TITLE
MPCDF update

### PR DESCRIPTION
**Pull request by dschwoerer**

### CODE REVIEW

File: bin/bout-build-deps.sh [hunk 1]
```diff
@@ -98,7 +98,7 @@ netcdf() {
 
 nccxx() {
     cd $BUILD
-    wget -c ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-cxx4-$NCCXXVER.tar.gz || :
+    wget -c https://downloads.unidata.ucar.edu/netcdf-cxx/4.3.1/netcdf-cxx4-$NCCXXVER.tar.gz || :
```
Comment (ZedThree) [inline bin/bout-build-deps.sh:101]: Should the `4.3.1` be `$NCCXXVER`?

**Comment by ZedThree** [comment]
How much of this should be in `BOUT-configs` btw?

File: bin/bout-build-deps.sh (hunk 1 above)
Comment (dschwoerer) [reply bin/bout-build-deps.sh:101]:
```suggestion
    wget -c https://downloads.unidata.ucar.edu/netcdf-cxx/$NCCXXVER/netcdf-cxx4-$NCCXXVER.tar.gz || :
```

**Comment by dschwoerer** [comment]
> How much of this should be in `BOUT-configs` btw?

I did look at BOUT-configs, but the description says it is all spack configs - and I do not have the time to learn something completely new.

**Comment by ZedThree** [comment]
It's a slightly confusing setup that we should get round to fixing, but there's actually a branch per machine.

**Review by bendudson — APPROVED** [review]

**Pull request merged**